### ISSUE TITLE
Install Tool does not work in dev environment

### ISSUE THREAD

**Issue by fritzmg** [contao/installation-bundle#72]
## Reproduction

1. Create a new Contao installation via `composer create-project contao/managed-edition`.
2. Create a fresh, empty database.
3. Open the install tool via `example.org/app_dev.php/contao/install`.
4. Accept the license, then enter a password.
5. Enter the database credentials and save the settings.

On the next step you will see the following error message:
```
Variable "sql_message" does not exist.
```
This is because the Twig template [`main.html.twig`](https://github.com/contao/installation-bundle/blob/4.4.8/src/Resources/views/main.html.twig) uses
```twig
{% if sql_message is not empty %}
```
However, at this point `sql_message` is a non-existing variable. It would be defined [here](https://github.com/contao/installation-bundle/blob/4.4.8/src/Controller/InstallationController.php#L390) - but in a fresh installation the function returns [here](https://github.com/contao/installation-bundle/blob/4.4.8/src/Controller/InstallationController.php#L360-L362) - thus the variable is never defined for the Twig rendering context.

Using `is not empty` on non-existent variables will lead to the `Variable "…" does not exist` error in the __dev__ environment (it will not yield this error in the __prod__ environment). See also https://straightupcraft.com/articles/testing-if-something-exists-is-defined-length-is-not-null-is-not-empty

You should use
```twig
{% if sql_message|default %}
```
instead.

**Comment by Toflar** [comment]
`{% if sql_message is defined %}` :)

**Comment by fritzmg** [comment]
> {% if sql_message is defined %} :)

That's not enough. You want to check if `sql_message` is defined and not empty in this case - since it could be defined, but empty. hence
```twig
{% if sql_message|default %}
```

**Comment by leofeyer** [comment]
Why not add the variable to the default context?

https://github.com/contao/installation-bundle/blob/264566ecdc70cd9b9b3a0518fac05ba582396daa/src/Controller/InstallationController.php#L45

**Comment by leofeyer** [comment]
Fixed in cd68833f39bf35f013e067dd49a4649188a19778.
